tests/models/test_cpu.py

```diff
@@ -218,12 +240,23 @@ def test_running_test_after_fitting(tmpdir):
     trainer.test()
 
     # test we have good test accuracy
-    tutils.assert_ok_model_acc(trainer, thr=0.5)
+    tutils.assert_ok_model_acc(trainer, key='test_loss', thr=0.5)
 
 
 def test_running_test_no_val(tmpdir):
     """Verify `test()` works on a model with no `val_loader`."""
-    model = EvalModelTemplate()
+    class ModelTrainTest(BoringModel):
+        def __init__(self, *args, **kwargs):
+            super().__init__(*args, **kwargs)
+
+        def val_loader(self):
+            pass
```

Comment:
Is this test relevant anymore? BoringModel does not have a `val_loader` attribute. Is it supposed to be `val_dataloader`?

Reply:
Doesn't seem fixed :thinking: 

Also update the docstring!

Reply:
@carmocca fixed it! Thank you :)